stopify/src/compile.ts

```diff
@@ -60,7 +60,7 @@ commander.arguments('<srcPath> <dstPath>');
 const args = commander.parse(process.argv);
 
 stopify(args.args[0], {
-  debug: args.boolean,
```

Comment:
This would have broken all the debugging languages, I guess the stopify bundle hasn't been redeployed with this change yet?